enterprise_subsidy/apps/api_client/enterprise_catalog.py

```diff
@@ -40,6 +42,8 @@ def get_content_metadata_for_customer(self, enterprise_customer_uuid, content_id
             content_identifier (str): **Either** the content UUID or content key identifier for a content record.
                 Note: the content needs to be owned by a catalog associated with the provided customer else this
                 method will throw an HTTPError.
+            skip_customer_fetch (bool): Forces enterprise-catalog to skip a sub-call to an edx-enterprise
+                API endpoint running in the edx-platform runtime. Defaults to True.
```

Comment:
[clarification] This comment suggests the default is `True`, but the function argument has the default set as `False`.